### PR TITLE
A MIMIC -> Piton tutorial

### PR DESCRIPTION
**Pull request by Lalaland**
This is a first draft of a MIMIC -> Piton tutorial.

The goal was as simple as an ETL as possible with minimal data loss.

The main difference between this and a real ETL is that a real ETL would try to actually map things to OMOP standard concepts. This does exactly zero mapping.

### CODE REVIEW

**Comment by Lalaland** [comment]
Closes #80

**Comment by Lalaland** [comment]
Also will close #61

File: native/database.cc [hunk 1]
```diff
@@ -74,7 +75,7 @@ void write_patient_to_buffer(uint32_t start_unique,
                              std::vector<uint32_t>& buffer) {
     if (current_patient.birth_date < epoch) {
         throw std::runtime_error(
-            absl::StrCat("Cannot have a birth date before epoch (1900) ",
+            absl::StrCat("Cannot have a birth date before epoch (1800) ",
```
Comment (jason-fries) [inline native/database.cc:78]: Can we add some comments here on why these values are chosen? A new person walking in will have zero idea why 1800 was chosen as a sensible minimum birthdate.

File: src/piton/etl_pipelines/omop.py [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+    parser.add_argument(
+        "--num_threads",
+        type=int,
+        help="The number of threads to use",
+        default=1,
+    )
+
+    args = parser.parse_args()
+
+    soft, hard = resource.getrlimit(resource.RLIMIT_NOFILE)
```
Comment (jason-fries) [inline src/piton/etl_pipelines/omop.py:62]: Again, some code comments would go a long way. What are these doing? Why do we set them?

File: native/database.cc [hunk 3]
```diff
@@ -20,13 +20,14 @@
...
-constexpr absl::CivilDay epoch(1900);
+constexpr absl::CivilDay epoch(1800);
+constexpr absl::CivilDay legacy_epoch(1900);
 
 constexpr int seconds_per_minute = 60;
 constexpr int minutes_per_hour = 60;
 constexpr int hours_per_day = 24;
 
-constexpr uint32_t current_version = 2;
+constexpr uint32_t current_version = 3;
```
Comment (jason-fries) [inline native/database.cc:30]: Will this branch support prior versions? for example, can I keep using the v7 extract after merging this branch into main?

**Review by jason-fries — CHANGES_REQUESTED** [review]
@Lalaland See some clarification question on version support.

**Pull request merged**